lighthouse-core/gather/gatherers/styles.js

```diff
@@ -129,13 +133,17 @@ class Styles extends Gatherer {
   afterPass(options) {
     return this.endStylesCollect(options.driver)
       .then(stylesheets => {
-        // Want unique stylesheets. Remove those with the same text content.
+        // Generally want unique stylesheets. Mark those with the same text content.
```

Comment:
IMO no, because the author is writing the `<style>` once and creating instances of the custom element. IOW, I don't think it's helpful to know you're failing 100 times. You really just want to know the source of failure, then fix that.